app/src/main/res/layout/main_login.xml

```diff
@@ -1,14 +1,55 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+    <data class="LoginFragmentBinding">
+        <variable
+            name="model"
+            type="com.dsu.poc.project.ui.login.model.LoginModel" />
 
-    <FrameLayout
+        <variable
+            name="viewModel"
+            type="com.dsu.poc.project.ui.login.LoginViewModel"/>
+
+    </data>
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_weight="55"
-        android:id="@+id/container_basic_login">
+        android:layout_height="match_parent">
+        <EditText
+            android:id="@+id/editUserName"
+            style="@style/Widget.AppCompat.EditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_centerVertical="true"
+            android:gravity="center_vertical"
+            android:layout_margin="10dp"
```

Comment:
please don't use hardcoded dimensions. They should be extracted from dimens.xml file.